content/en/docs/ibm/create-cluster.md

```diff
@@ -128,12 +189,150 @@ ibmcloud ks cluster create ${WORKER_NODE_PROVIDER} \
   --public-vlan ${PUBLIC_VLAN_ID} 
 ```
 
-Wait until the cluster is deployed and configured. It can take a while for the cluster to be ready. Run with following command to periodically check the state of your cluster. Your cluster is ready when the state is `normal`.
+Wait until the cluster is deployed and configured. It can take a while for the cluster to be ready. Run with following
+command to periodically check the state of your cluster. Your cluster is ready when the state is `normal`.
 
 ```shell
 ibmcloud ks clusters --provider ${WORKER_NODE_PROVIDER} |grep ${CLUSTER_NAME}|awk '{print "Name:"$1"\tState:"$3}'
 ```
 
+## Create an IBM Cloud Kubernetes cluster for `vpc-gen2` infrastructure
+
+Creating a `vpc-gen2` based cluster needs a VPC, a subnet and a public gateway attached to it. Fortunately, this is a one
+time setup. Future `vpc-gen2` clusters can reuse the same VPC/subnet(with attached public-gateway). 
+
+1. Begin with installing a vpc-infrastructure plugin:
+
+   `$ ibmcloud plugin install vpc-infrastructure`
+   
+   Refer to this [link](https://cloud.ibm.com/docs/containers?topic=containers-vpc_ks_tutorial), for more information.
+
+2. Target the gen 2 to access gen 2 resources:
+
+   `$ ibmcloud is target --gen 2`
+   
+   Verify, the target is correctly setup
+
+   ```shell
+   $ ibmcloud is target
+   Target Generation: 2
+   ```
+
+3. Create or use an existing VPC:
+   
+   a) Use an existing VPC: 
+   
+    ```shell
+    
+    $ ibmcloud is vpcs
+    Listing vpcs for generation 2 compute in all resource groups and region ...
+    ID                                          Name                Status      Classic access   Default network ACL                                    Default security group                                 Resource group   
+    r006-hidden-68cc-4d40-xxxx-4319fa3gxxxx   my-vpc1              available   false            husker-sloping-bee-resize                              blimp-hasty-unaware-overflow                           kubeflow   
+    
+    ```
```

Comment:
```suggestion
    ```shell
    ibmcloud is vpcs
    ```

    Example output:
    ```
    Listing vpcs for generation 2 compute in all resource groups and region ...
    ID                                          Name                Status      Classic access   Default network ACL                                    Default security group                                 Resource group   
    r006-hidden-68cc-4d40-xxxx-4319fa3gxxxx   my-vpc1              available   false            husker-sloping-bee-resize                              blimp-hasty-unaware-overflow                           kubeflow   
    ```
```

Careful with the placement of ```, since it may affect the rendering in the entire Markdown file